src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1452,6 +1508,44 @@ private void PlaceIfPossible(string organelleType, int q, int r, int rotation, r
         }
     }
 
+    /// <summary>
+    ///   A recursive function that adds my neighbors to the checked list and calls the function for each neighbor.
+    /// </summary>
+    /// <param name="checked">
+    ///   The list of visited hexes will be filled up.
+    /// </param>
+    /// <param name="me">
+    ///   Which neighbors to visit next.
+    /// </param>
+    private void CheckmarkNeighbors(List<Hex> @checked, Hex me)
+    {
+        // Get all neighbors not already visited
+        var myNeighbors = GetNeighborHexes(me).Where(p => !@checked.Contains(p)).ToArray();
+
+        // Add the new neighbors to the list
+        @checked.AddRange(myNeighbors);
+        foreach (var neighbor in myNeighbors)
+        {
+            CheckmarkNeighbors(@checked, neighbor);
+        }
+    }
+
+    /// <summary>
+    ///   Gets all neighboring hexes
+    /// </summary>
+    /// <param name="hex">
+    ///   The hex to get the hexes for
+    /// </param>
+    /// <returns>
+    ///   Returns a list of neighbors
+    /// </returns>
```

Comment:
These should be single line elements to save space on this not so important method.